.github/workflows/rego-tests.yaml

```diff
@@ -2,7 +2,11 @@ name: test regos
 on:
   pull_request:
     branches: [dev, master]
-    types: [opened]
+    types: [ edited, opened, synchronize, reopened ]
+    paths-ignore:
+      # Do not run the pipeline if only Markdown files changed
+      - '**.yaml'
```

Comment:
did you purposely add the yaml here?

Reply:
yes, but an updated version of this doc is going to be implemented very soon (yaml is included in it)

Reply:
new version contains only **md ignore path's